### ISSUE TITLE
lief.ELF.ARCH.None yields invalid syntax

### ISSUE THREAD

**Issue by bdemick** [lief-project/LIEF#745]
**Describe the bug**
In Python, using lief to check for `machine_type` being set to "No machine" by comparing the machine_type to `lief.ELF.ARCH.None` results in an invalid syntax error. Per the documentation, this should be the valid object when the `e_machine` value for an ELF is set to 0 (No machine). 

**To Reproduce**
```
import lief 

f = lief.parse("test.elf")
if f.header.machine_type == lief.ELF.ARCH.AARCH64: # This is fine
    print("AARCH64")
elif f.header.machine_type == lief.ELF.ARCH.None: # Invalid syntax
    print("No machine")
```

**Expected behavior**
A working enumeration value for `No machine`.

**Environment (please complete the following information):**
 - System and Version : macOS 12.4
 - Target format: ELF
 - LIEF commit version: 0.12.1-c7b3ce3b

**Additional context**
My guess is this has something to do with `.None` involving a reserved keyword in Python, but I'm not exactly sure of the exact rule. I'd think this would work fine if "None" were changed "NONE", but have not tested it.

**Comment by romainthomas** [comment]
Indeed, `None` is a reserved keyword. I'll fix it

**Comment by romainthomas** [comment]
Fixed with 64ead975b8828774fafe9b097ac4b0d05b184065

**Comment by bdemick** [comment]
Right on. Thanks!
